### DISCUSSION TITLE
Content Editor - small menu

### DISCUSSION THREAD

**Discussion by ceyerkes** [twilio-labs/paste#3124] [General] [answered]
Hello hellooo!

I don't know what to call this, but in a couple areas of the new content editor, we need a small menu:

![Screenshot 2023-03-27 at 9 16 25 AM](https://user-images.githubusercontent.com/87141551/227984678-2a02b57b-c85a-47d2-bb42-48b2e36cb988.png)
(after selecting an image in the RTE)

![Screenshot 2023-03-27 at 9 17 04 AM](https://user-images.githubusercontent.com/87141551/227984894-4e9dcaa8-ea40-4c3a-a904-eef6a8d54b78.png)

![Screenshot 2023-03-27 at 9 17 47 AM](https://user-images.githubusercontent.com/87141551/227985146-2fbf619e-b39a-4863-b0c1-6ae3ea500a44.png)

![Screenshot 2023-03-27 at 9 17 29 AM](https://user-images.githubusercontent.com/87141551/227985036-540c821f-a676-4ee4-b1ce-9a6765d599fa.png)
(a few places in the RTE menu itself)

Wondering if I should keep using the combobox row that I pulled out for this use case, or if there's another suggestion. Happy to submit a dependency ticket if that's preferred as well. 

Thanks!

**Comment by loreina** [comment, marked as answer]
Hi @ceyerkes,

You can create this horizontal icon menu by using our [Menu Primitive](https://paste.twilio.design/primitives/menu-primitive). 

For the menu trigger button, you could use 2 icons with the chevron icon at our smallest icon size `sizeIcon05`. For the menu items, I'd recommend copying our [toggle button styles](https://paste.twilio.design/components/button#toggle-button). It might be easier to see those styles in [Figma](https://www.figma.com/file/l2qtpQVMmXzQPVB3rEAA8i/%5BNEW%5D-Paste-Components?node-id=21000-142068&t=Cw1rboJDSmUHjwoF-4).

Feel free to drop by office hours with your engineer and our team can help with the design and build. Thanks!
